README.md

```diff
@@ -20,59 +20,60 @@
   </a>
 </p>
 
+*Spring Cloud Data Flow* is a microservices-based streaming and batch data processing pipelines in Cloud Foundry and
+Kubernetes.
 
-*Spring Cloud Data Flow* is a toolkit for building data integration and real-time data processing pipelines. 
-
-Pipelines consist of Spring Boot apps, built using the [Spring Cloud Stream](https://github.com/spring-cloud/spring-cloud-stream)
+Data processing pipelines consist of Spring Boot apps, built using the [Spring Cloud Stream](https://github.com/spring-cloud/spring-cloud-stream)
 or [Spring Cloud Task](https://github.com/spring-cloud/spring-cloud-task) microservice frameworks. 
 
-This makes Spring Cloud Data Flow suitable for a range of data processing use cases, from import/export to 
-event streaming and predictive analytics.
+This makes Spring Cloud Data Flow suitable for a range of data processing use cases, from import/export to event streaming
+and predictive analytics.
 
 ----
 
 ## Components
 
-The [Core](https://github.com/spring-cloud/spring-cloud-dataflow/tree/master/spring-cloud-dataflow-core)
-domain module includes the concept of a *stream* that is a composition of spring-cloud-stream
-modules in a linear pipeline from a *source* to a *sink*, optionally including *processor* application(s)
-in between. The domain also includes the concept of a *task*, which may be any process that does
-not run indefinitely, including [Spring Batch](https://github.com/spring-projects/spring-batch) jobs.
-
-The [App Registry](https://github.com/spring-cloud/spring-cloud-dataflow/tree/master/spring-cloud-dataflow-registry)
-maintains the set of available apps, and their mappings to URIs.
-For example, if relying on Maven coordinates, an app's URI would be of the format:
-`maven://<groupId>:<artifactId>:<version>`
+**Architecture**: The Data Flow Server is a Spring Boot application that provides REST APIs, UI, and Java DSL.
```

Comment:
We can probably say, 

> The Data Flow Server is a Spring Boot application that exposes RESTful API and REST clients (Shell, Dashboard UI, Java DSL) to interact with the API.

Reply:
Following the comment below:
A Data Flow installation consists of a Data Flow Server and Skipper Server[link]. These are Spring Boot Applications that provide RESTful APIs. The Data Flow Server provides a Dashboard UI.  Additionally, Shell applications,  and a Java DSL are available to interact with the API.  These services are backed by an RDMS which is typically a dedicated server, but may also be an existing installation.

Reply:
This is too verbose. I was hoping to keep it tight and make folks click the architecture and feature links to read more about the offering.

Reply:
That's better

Reply:
Yes indeed. This is crisp. A couple of suggestions:

 - I think we need `Spring Cloud Data Flow Server` in all the places. 
 - Instead of just `deploying`, maybe we can bring back our term `orchestrating` or simply `managing`

Reply:
Hmm, we use Data Flow in the ref. docs and microsite liberally, as a short form. I can change it here, but then everywhere else it will be inconsistent.

IMO, though, deploying defines it more explicitly. The "manage" aspect maybe applies to Tasks, but not necessarily entirely applicable.

Would you still recommend these changes?

Reply:
I would vote for consistency across the board for `Spring Cloud Data Flow` instead of `Data Flow` but at least for now, inside the README here as it should not be confused with some kind of integration to Google `Dataflow`. 

On the `manage` vs `deploy` part, what do you think about `orchestrating` as it kind of applies to both streams and tasks? The word `deploying` seems to suggest just the **deployment** of the streams and tasks.